docs/source/reservations/reservations.rst

```diff
@@ -235,7 +235,7 @@ When using DWS with Kueue, you can additionally specify `max_run_duration`. With
     kueue:
       local_queue_name: skypilot-local-queue
 
-Or, specify it in the SkyPilot YAML (see :ref:`here <config-client-job-task-yaml>`):
+Or, specify the ``dws`` and ``kueue`` fields in the SkyPilot YAML (see :ref:`here <config-client-job-task-yaml>`):
```

Comment:
It's a bit too hard to read and understand that config must be split between task.yaml (for dws and kueue fields) and  config.yaml (for autoscaler field). 

Can we keep it simple and remove L238-263?